### PR TITLE
add emoji support via markdown-it-emoji plugin

### PR DESCRIPTION
**Pull request by ashleygwilliams**
replacing #60 cc @patriciarealini

### CODE REVIEW

File: test/fixtures/github.md [hunk 1]
```diff
@@ -15,6 +15,8 @@
 ![](//protocollie.com/woof.png)
 ![](http://insecure.com/bad.png)
 ![](https://secure.com/good.png)
+
+<<<<<<< HEAD
```
Comment (zeke) [inline test/fixtures/github.md]: It looks like some git conflict junk got left behind

Comment (ashleygwilliams) [inline test/fixtures/github.md]: lol yeah jfc the rebase for this was intense. surprised i didn't miss more.

File: lib/render.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ var Highlights = require('highlights')
 var MD = require('markdown-it')
 var lazyHeaders = require('markdown-it-lazy-headers')
 var cleanup = require('./cleanup')
+var emoji = require('markdown-it-emoji')
+var MD = require('markdown-it')
```
Comment (wooorm) [inline lib/render.js]: Double import of markdown-it, more specifically, L7 is a duplicate of L3

Comment (ashleygwilliams) [inline lib/render.js]: good catch! thanks, that rebase was brutal

**Comment by ashleygwilliams** [comment]
@aredridel  :sparkles:review plz :sparkles:

p.s. @patriciarealini czech out this patch to see the test i wrote :)

**Comment by jefflembeck** [comment]
:+1:

**Comment by ashleygwilliams** [comment]
thanks @jefflembeck :dolphin: :space_invader: ~~ merging

**Comment by patriciarealini** [comment]
:clap::clap::clap::clap::clap::clap:

**Comment by patriciarealini** [comment]
closes https://github.com/npm/marky-markdown/issues/59

**Comment by patriciarealini** [comment]
zats beautiful, thank you @ashleygwilliams

**Pull request merged**